OasisDEX/oasis-borrow
# Gf/stoploss confirmation screen

# [Confirmation Screens for auto buy, sell and stop loss](https://app.shortcut.com/oazo-apps/story/5353/stop-loss-add-order-summary-confirmation-screen)
  
## Changes 👷‍♀️
- Added Confirmation screens to auto buy, sell and stop loss flows
- Added step numbers
- Added edit order button
  
## How to test 🧪
- Go through the auto buy, sell and stop loss set up flows. You will see a new confirmation screen and added step numbers to the flow. 
- On the confirmation screen an edit order button should now also be present, when clicked it should take you back to the previous step.
